apps/glusterfs/operations_device.go

```diff
@@ -1033,3 +1016,29 @@ func getWorkingNode(n *NodeEntry, db wdb.RODB, executor executors.Executor) (str
 	}
 	return node, err
 }
+
+func tryDestroyBrickMap(bhmap brickHostMap, executor executors.Executor) (ReclaimMap, error) {
+	reclaimed, err := bhmap.destroy(executor)
+	if err == nil {
+		return reclaimed, nil
+	}
+
+	// If the destroy failed because the node is not running (ideally,
+	// permanently down) we want to continue with the operation . But if
+	// the node works we failed for a "real" reason we don't want to
+	// blindly ignore errors. So we do a probe to see if the host is
+	// responsive. If the destroy has failed we will use the node's
+	// responsiveness to decide if // this should be treated as a failure
```

Comment:
here also **//** can be removed in b/w the line

Reply:
Fixed. Thanks.